src/helpers/findAll.ts

```diff
@@ -12,8 +13,11 @@ export function findAll(
   options?: FindAllOptions
 ) {
   const results = root.findAll(predicate);
+  const includeHiddenQueryOption = options?.includeHidden ?? options?.hidden;
+  const defaultIncludeHidden =
+    getConfig()?.defaultIncludeHidden ?? getConfig()?.defaultHidden;
 
-  const hidden = options?.hidden ?? getConfig().defaultHidden;
+  const hidden = includeHiddenQueryOption ?? defaultIncludeHidden;
```

Comment:
```
const includeHidden = 
  options?.includeHidden ??
  options?.hidden ??
  getConfig()?.defaultIncludeHidden ??
  getConfig()?.defaultHidden
 }

```

Reply:
i wanted to avoid nested operators to maximise readibility, do you find it more readable your way?

Reply:
Yes, I find it much more readable, as a single nullish coalescing operator. Having it as separate intermediate vars would be useful if it used a mix of two operators (like `&&` and `||`) with different precedence, etc. In case all vars use the same operator, I found putting it together removes the question why are there some intermediate variables :-)